app/models/token_sender.rb

```diff
@@ -1,7 +1,7 @@
 class TokenSender
   def initialize(user, options={})
```

Comment:
Whenever we have time for a stand-up, I'd also like to go through how Twilio does it's job.